pages/linux/cryptsetup.md

```diff
@@ -4,8 +4,8 @@
 
 - Format a drive (or a file) to make it a LUKS volume:
 
-`cryptsetup luksFormat {{path_to_luks_volume_to_be}}`
+`cryptsetup luksFormat {{path/to/luks_volume_to_be}}`
 
 - Open a volume so that you can access it decrypted on the fly at /dev/mapper/{{a}}:
 
-`cryptsetup luksOpen {{path_to_luks_volume_to_be}} {{a}}`
+`cryptsetup luksOpen {{path/to/luks_volume_to_be}} {{a}}`
```

Comment:
These comments seem to be about the `cryptsetup` command. I need to research this a bit. How about I push this commit and address these concerns in a separate commit ?